src/app/api.js

```diff
@@ -232,6 +232,25 @@ const api = {
                 console.error(error)
                 return false
             })
+    },
+    csv: async (access_token, query_parameters) => {
+        return await axios(`${url}/csv`, {
+            method: "POST",
+            headers: {
+                authorization: `BEARER ${access_token}`,
+                "Content-Type": "application/json",
+                // allow caching of data
+                "Cache-Control": "private"
+            },
+            data: query_parameters
```

Comment:
```suggestion
            data: query_parameters,
            responseType: "blob"
```

Needed for IE I think.